SpatialGDK/Source/SpatialGDKEditor/SpatialGDKEditor.Build.cs

```diff
@@ -13,9 +13,13 @@ public SpatialGDKEditor(ReadOnlyTargetRules Target) : base(Target)
 			new string[] {
 				"Core",
 				"CoreUObject",
+                "EditorStyle",
```

Comment:
Indentation looks wrong.

Reply:
It looks like the new changes uses spaces where the old ones use tabs. Could you replace it with tabs to be consistent with the rest of the file?